docs_outdoornav_user_manual/features/sensor_filters.mdx

```diff
@@ -0,0 +1,99 @@
+---
+title: Sensor Filters
+sidebar_label: Sensor Filters
+sidebar_position: 2
+toc_min_heading_level: 2
+toc_max_heading_level: 4
+---
+
+The following table lists the sensors that are useable with the
+OutdoorNav software. These sensor drivers can be turned on/off using
+these environment variables, however, the sensor will always remain
+powered on.
+
+| Environment Variable           | Description                               | Data Type
+|--------------------------------|-------------------------------------------|--------------
+| **SWIFTNAV_ENABLE_DRIVER**     | Enable/disable the Swiftnav Piksi/Duro ROS driver, if integrated. | bool |
+| **UBLOX_ENABLE_DRIVER**        | Enable/disable the UBlox ROS driver, if integrated. | bool |
+| **MICROSTRAIN_ENABLE_DRIVER**  | Enable/disable the Microstrain GX5/CV5 ROS driver, if integrated.  | bool |
+| **XSENS_ENABLE_DRIVER**        | Enable/disable the XSens MTI ROS driver, if integrated.  | bool |
+| **VLP_ENABLE_DRIVER**          | Enable/disable the Velodyne ROS driver, if integrated (front unit if more than one).  | bool |
+| **REAR_VLP_ENABLE_DRIVER**     | Enable/disable the Velodyne ROS driver, if integrated (rear unit if more than one).  | bool |
+| **LMS1XX_ENABLE_DRIVER**       | Enable/disbale the Sick LMS1XX ROS driver, if integrated (front unit if more than one).  | bool |
+| **REAR_LMS1XX_ENABLE_DRIVER**  | Enable/disbale the Sick LMS1XX ROS driver, if integrated (rear unit if more than one).  | bool |
+| **HOKUYO_ENABLE_DRIVER**       | Enable/disbale the Hokuyo ROS driver, if integrated (front unit if more than one).  | bool |
+| **REAR_HOKUYO_ENABLE_DRIVER**  | Enable/disbale the Hokuyo ROS driver, if integrated (rear unit if more than one).  | bool |
+| **D435_ENABLE_DRIVER**         | Enable/disable the Realsense ROS driver, if integrated (front unit if more than one).  | bool |
+| **REAR_D435_ENABLE_DRIVER**    | Enable/disable the Realsense ROS driver, if integrated (rear unit if more than one).  | bool |
+
+
+### Sensor Tuning (Advanced)
+
+The following list of environment variables can be used to filter the
+sensor data in order to modify/improve the sensory input to the
+OutdoorNav autonomy. It is recommended that you first consult the
+following links before attempting to modify these parameters.

```

Comment:
If setting these incorrectly could cause safety issues, we should have a WARNING box.